### PR TITLE
Browser tests

### PR DESCRIPTION
**Pull request by KeeganWoods1**
##### _What existing problem does the pull request solve?_

Tests for viewing booking request status, restricting roles from rooms and editing a pending booking

##### _List all issues this PR resolves_
- closes #XXXX

##### _Complete these items before requesting reviews_
- [ ] Static Analysis Rules Validated
- [ ] All commits follow naming conventions

### Sub-tasks Checklist
- [ ] UI Implementation
- [ ] UI Test
- [ ] Backend Implementation
- [ ] Backend Tests
- [ ] Integration Tests
- [ ] System Tests
- [ ] Documentation

### CODE REVIEW

File: tests/Browser/BookingsPageTest.php [hunk 1]
```diff
@@ -100,4 +100,44 @@ public function testClickOnReviewBooking()
...
+        $bookings = BookingRequest::factory()
+            ->count(1)
+            ->hasReservations(random_int(1, 3))
+            ->create(["status" => BookingRequest::PENDING])->first();
+
+        $this->browse(function (Browser $browser) use ($bookings) {
+            $browser->loginAs(User::first());
+            $browser->visit('/bookings')
+                ->clickLink("View", 'button')
+                ->pause(5000);
```
Comment (alexstojda) [inline tests/Browser/BookingsPageTest.php:114]:
```suggestion
                ->pause(250);
```
5 seconds is a longggggg time. If every test had a 5 second delay, CI would take hours! :p 

Try with a small delay and work your way up

File: tests/Browser/BookingsPageTest.php [hunk 2]
```diff
@@ -100,4 +100,44 @@ public function testClickOnReviewBooking()
...
+    {
+        $bookings = BookingRequest::factory()
+            ->count(1)
+            ->hasReservations(1)
+            ->create(["status" => BookingRequest::PENDING])->first();          
+        $this->browse(function (Browser $browser) use ($bookings) {
+            $browser->loginAs(User::first());
+            $browser->visit('/bookings')
+                ->clickLink("Edit", 'button')
+                ->pause(5000);
```
Comment (alexstojda) [inline tests/Browser/BookingsPageTest.php:129]:
```suggestion
                ->pause(250);
```

File: tests/Browser/BookingsPageTest.php [hunk 3]
```diff
@@ -100,4 +100,44 @@ public function testClickOnReviewBooking()
...
+            $browser->assertSee('Submit A Booking Request')
+                ->type('@title', 'title')
+                ->type('@type', 'type')
+                ->type('#event_description', 'description')
+                ->type('#guest_speakers', 'speakers')
+                ->type('@numberAttending', '2');
+            $browser->check('div > main > form > div:nth-child(4) > div > div > input');
+            $browser->press('SUBMIT')
+                ->pause(5000)
```
Comment (alexstojda) [inline tests/Browser/BookingsPageTest.php:138]:
```suggestion
                ->pause(250)
```

**Review by alexstojda — CHANGES_REQUESTED** [review]
smol things

**Review by alexstojda — APPROVED** [review]

**Review by EvanDime — APPROVED** [review]

**Pull request merged**